src/quo2/components/selectors/selectors.cljs

```diff
@@ -1,109 +1,73 @@
 (ns quo2.components.selectors.selectors
   (:require [quo2.components.icon :as icons]
-            [quo2.foundations.colors :as colors]
+            [quo2.components.selectors.styles :as style]
             [react-native.core :as rn]
-            [reagent.core :as reagent]
-            [quo2.components.selectors.styles :as style]))
+            [reagent.core :as reagent]))
 
 (defn- handle-press
-  [disabled? on-change checked?]
-  (when (not disabled?)
-    (fn []
-      (swap! checked? not)
-      (when on-change (on-change @checked?)))))
+  [on-change checked-atom checked?]
+  (when checked-atom (swap! checked-atom not))
+  (when on-change (on-change (not checked?))))
 
-(defn checkbox-prefill
-  [{:keys [default-checked?]}]
-  (let [internal-checked? (reagent/atom (or default-checked? false))]
-    (fn [{:keys [on-change disabled? blurred-background? container-style checked?]}]
-      (when (and (not (nil? checked?)) (not= @internal-checked? checked?))
-        (reset! internal-checked? checked?))
-      (let [checked? (or checked? @internal-checked?)]
+(defn- selector
+  [{:keys [default-checked? checked?]}]
+  (let [controlled-component? (some? checked?)
+        internal-checked?     (when-not controlled-component?
+                                (reagent/atom (or default-checked? false)))]
+    (fn [{:keys [checked? disabled? blur? custom-color on-change container-style label-prefix
+                 outter-style-fn inner-style-fn icon-style-fn]
+          :or   {custom-color :blue}}]
```

Comment:
It receives `custom-color` now,

Reply:
this should be `customization-color` for consistency - https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#custom-colors

Reply:
Thanks! :sweat_smile: I forgot the correct name. Solved in 9dbb90b931175569414dace3e7cd825e8138abb1